server/models/CollectionJob.js

```diff
@@ -9,7 +9,8 @@ module.exports = function (sequelize, DataTypes) {
             id: {
                 type: DataTypes.STRING,
                 allowNull: false,
-                primaryKey: true
+                primaryKey: true,
+                autoIncrement: true
```

Comment:
Wouldn't this need a migration?

Reply:
Oh shoot this change snuck in from another PR, I will remove it.